### ISSUE TITLE
 Unable to locate a class or view for component [infocalloutComponent]. 

### ISSUE THREAD

**Issue by SMEWebify** [SMEWebify/WebErpMesv2#9]
Thank for your great work, several of this issues are now solved, there is another issues yet, that I think is correlated to this one.

In some pages I encounter this fatal error
```
InvalidArgumentException
Unable to locate a class or view for component [infocalloutComponent]. 
```

For example I find this error in the kanban page `/production/kanban`

I not able to understand if it is always correlated to missmatch case. It looks that the code is right. This is the first project in Laravel I ever study, sorry :(

_Originally posted by @RobertoBochet in https://github.com/billyboy35/WebErpMesv2/issues/7#issuecomment-1017570464_

**Comment by SMEWebify** [comment]
@RobertoBochet  

infocalloutComponent is called when I use a blade component that allows me to put help at the top of the page.

The class blade:
app/View/Components/infocalloutComponent.php

```
  public function render()
     {
         return view('components.infocallout-component');
     }
```

call the view: (verify that you have in folder )
resources/views/components/infocallout-component.blade.php

--

at the time of writing, I don't see a problem with case sensitive
I guess you must have other error on other page (like you company, or methods?)

--

Aslo this page executes a vue.js component did you execute in terminal ?
npm install
npm run dev

--

I keep looking :s

**Comment by RobertoBochet** [comment]
> Aslo this page executes a vue.js component did you execute in terminal ?
> npm install
> npm run dev

I used #8 to perform my test, it includes `npm ci` and `composer install`. If you want you can use it to reproduce the bug

I want to be more useful but as I said I am a beginner in Laravel

> I guess you must have other error on other page (like you company, or methods?)

All the error I see are related to that component

**Comment by SMEWebify** [comment]
I may have found a way out.

My blade component class did not start with a capital letter. What does not meet the standards

https://stackoverflow.com/questions/68494441/laravel-8-unable-to-locate-a-class-or-view-for-component-dashboardheader

can you test (while waiting for me to learn docker ^^)? I just updated the files

I don't know Docker too well, for the moment I can't mount an image, it returns an error and goes to a recovery container

**Comment by RobertoBochet** [comment]
Your fix partially works, I need to rename also the component file. I included this fix in #12

**Comment by RobertoBochet** [comment]
Sorry for docker, I assumed you can run it.

If you are able to install `docker-compose` you should only create `.env` file (the laravel .env file) then run the command `docker-compose up` you should be able to reach the app at address `http://127.0.0.1:45060`,  docker will do all the magic 😉

NB issue #11 can create you some problem with docker
